tests/src/limits/ThrottleTests.scala

```diff
@@ -55,13 +55,27 @@ class ThrottleTests
     val defaultAction = Some(TestUtils.getTestActionFilename("hello.js"))
 
     val throttleWindow = 1.minute
-    val maximumInvokesPerMinute = WhiskProperties.getProperty("limits.actions.invokes.perMinute").toInt
-    val maximumFiringsPerMinute = WhiskProperties.getProperty("limits.triggers.fires.perMinute").toInt
-    val maximumConcurrentInvokes = WhiskProperties.getProperty("limits.actions.invokes.concurrent").toInt
+
+    val maximumInvokesPerMinute = getLimit("defaultLimits.actions.invokes.perMinute", "limits.actions.invokes.perMinute")
+    val maximumFiringsPerMinute = getLimit("defaultLimits.triggers.fires.perMinute", "limits.triggers.fires.perMinute")
+    val maximumConcurrentInvokes = getLimit("defaultLimits.actions.invokes.concurrent", "limits.actions.invokes.concurrent")
+
+    println(s"maximumInvokesPerMinute  = $maximumInvokesPerMinute")
+    println(s"maximumFiringsPerMinute  = $maximumFiringsPerMinute")
+    println(s"maximumConcurrentInvokes = $maximumConcurrentInvokes")
 
     val rateMessage = "Too many requests from user"
     val concurrencyMessage = "The user has sent too many requests in a given amount of time."
 
+    /*
+     * Retrieve a numeric limit for the key from the property set.  If the overrideKey is present, use that.
+     */
+    def getLimit(key: String, overrideKey: String) = try {
+        WhiskProperties.getProperty(overrideKey).toInt
+    } catch {
+        case _: Throwable => WhiskProperties.getProperty(key).toInt
+    }
```

Comment:
I think, using the "functional" way, a `Try` makes this more readable (and idiomatic).

```
def getLimit(key: String, overrideKey: String) = Try {
    WhiskProperties.getProperty(overrideKey).toInt
} getOrElse {
    WhiskProperties.getProperty(key).toInt
}
```